src/path_parsing.rs

```diff
@@ -1,37 +1,26 @@
+extern crate memchr;
+use self::memchr::memrchr;
 use std::path::MAIN_SEPARATOR;
+use std::str;
 
-static SEP: u8 = MAIN_SEPARATOR as u8;
+pub const SEP: u8 = MAIN_SEPARATOR as u8;
+// TODO: Do not hard-code "/". How can we convert MAIN_SEPARATOR char to a static string here?
```

Comment:
I think [`lazy_static`](https://crates.io/crates/lazy_static) crate can help with this one:

```rust
lazy_static! {
    pub static ref SEP_STRING: String = MAIN_SEPARATOR.to_string();
    pub static ref SEP_STR: &'static str = &SEP_STRING;
}
```

Reply:
Done, https://users.rust-lang.org/t/a-static-str-version-of-std-path-main-separator/14392/4.